test/spec/unit/core/adapterManager_spec.js

```diff
@@ -3,6 +3,9 @@ import AdapterManager from 'src/adaptermanager';
 import { getAdUnits } from 'test/fixtures/fixtures';
 import CONSTANTS from 'src/constants.json';
 import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+require('modules/appnexusAstBidAdapter');
```

Comment:
Accidental import?

Reply:
Not accidental, i took a shortcut to test old way of aliasing adapter. Since we will be only keeping this code for a short time. 

But anyway, updated test case to make it as it should be.

Reply:
ah ok...  no worries either way.